Index.html

```diff
@@ -0,0 +1,97 @@
+<!--DOCTYPE html-->
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Tribute Page - Oprah winfrey</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <div class="influential">
+
+            <div id="title">
+                <h1>Oprah Winfrey</h1>
+                <p> One of the most influential woman in the world </p>
+
+                
+        </div>
+        <div class="influential">
+            <main id="main">
+                <section class="container" id="img-div">
+                    <figure>
+                        <img src="https://allstarbio.com/wp-content/uploads/2017/01/Oprah-Winfrey-photos.jpg"
+                            alt="Oprah Winfrey" id="image" class="center">
+                        <figcaption id="img-caption">In Miami on The Life You Want tour, October 2014 </figcaption>
+                    </figure>
+    
+    
+                    <section class="container short-story" id="tribute-info">
+    
```

Comment:
You have used away too many IDs in your HTML please consider reducing them and replace those with a class